### DISCUSSION TITLE
Minimal reformat of the product addition

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-androidapp#3960] [Ideas]
![image](https://user-images.githubusercontent.com/1689815/53698366-09525180-3ddc-11e9-8241-769d033d8640.png)

**Comment by VaiTon** [comment]
It would fix #499.

**Comment by teolemon** [comment]
It should come with a warning that ingredients have been extracted when leaving without clicking on the green "Looks good" button for ingredients

**Comment by teolemon** [comment]
blocked on the merge of #3072

**Comment by teolemon** [comment]
#3072 has been merged. We can move ahead on this one.
